website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -0,0 +1,194 @@
+- [Run exports in production](#exports-in-production) using the [dbt Cloud job scheduler](/docs/deploy/job-scheduler).
+
+### Exports in development
+
+You can run an export in your development environment using your development credentials if you want to test the output of the export before production. You can use the following command in the dbt Cloud CLI:
+
+```bash
+dbt sl export
+```
+
+You can also run any export defined for the saved query and materialize the table or view in your development environment. For example: 
+
+```bash
+dbt sl export --saved-query sq_name
+Polling for export status - query_id: 2c1W6M6qGklo1LR4QqzsH7ASGFs..
+Export completed.
+```
+---- ADD OUTPUT OF THE COMMAND HERE ----
+
+### Use the select flag
+
+By default, all exports are run for a saved query. You can use the `select` flag in [development](#exports-in-development).
+
+For example, the following command runs `export_1` and `export_2` and doesn't work with the `--alias` or `--export_as` flags:
+
+```bash
+dbt sl export --saved-query sq_name --select export_1,export2
+```
+
+<details>
+<summary>Overriding export configurations</summary>
+
+The `--select` flag is mainly used to include or exclude specific exports. If you need to change these settings, you can use the following flags to override export configurations:
+
+- `--export-as` &mdash; Defines the materialization type (table or view) for the export. This creates a new export with its own settings and is useful for testing in development.
+- `--schema` &mdash;  Specifies the schema to use for the  materialized table or view.
+- `--alias` &mdash; Assigns a custom alias to the materialized table or view. This overrides the default export name.
+
+Be careful. The `--select` flag _can't_ be used with `alias` or `schema`.
+
+For example, you can use the following command to create a new export named `new_export` as a table:
+
+```bash
+dbt sl export --saved-query sq_number1 --export-as table --alias new_export
+```
+</details>
+
+### Exports in production
+
+To enable exports in production using the dbt Cloud job scheduler, follow these steps:
+
+**Set environment variable**
+
+1. Go to your **Environments** page by clicking **Deploy** in the navigation.
+2. Select **Environment variables**
+3. [Set the environment variable](/docs/build/environment-variables#setting-and-overriding-environment-variables) key to `DBT_INCLUDE_SAVED_QUERY` and the environment variable(s) value to `TRUE` (`DBT_INCLUDE_SAVED_QUERY=TRUE`).
+
+Doing this ensures saved queries and exports are included in your dbt build job. For example, running `dbt build sq_name` runs the equivalent of `dbt sl export --saved-query sq_name` in the dbt Cloud Job scheduler.
+
+If exports aren't needed, you can set the value(s) to `FALSE` (`DBT_INCLUDE_SAVED_QUERY=FALSE`).
+
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/deploy_exports.jpg" width="90%" title="Add an environment variable to run exports in your production run." />
```

Comment:
@JStein77 i updated teh screenshot so it's super clear that the key only contains `DBT_INCLUDE_SAVED_QUERY` and users don't make the same mistake i did